ftplugin/ps1xml.vim

```diff
@@ -14,6 +14,11 @@ let b:did_ftplugin = 1
 setlocal tw=0
 setlocal commentstring=#%s
 setlocal formatoptions=tcqro
+" Enable autocompletion of hyphenated PowerShell commands,
+" e.g. Get-Content or Get-ADUser
+setlocal iskeyword+=-
```

Comment:
Hi Heath,

This is the first time, I’ve submitted a PR that had issues with it so I’m still figuring out the GitHub review workflow. I’ve fixed the issue and pushed a new commit so I hope it’s OK now.

Thanks for your patience, understanding (and diligence),
Anthony